label_studio/ml_models/models.py

```diff
@@ -159,3 +168,17 @@ def output_file_name(self):
     @property
     def error_file_name(self):
         return f'{self.project.id}_{self.model_version.pk}_{self.pk}/error.csv'
+
+    def delete_predictions(self):
+        """
+        Deletes any predictions that have originated from a ModelRun
+        """
+
```

Comment:
does [updating the task counters](https://github.com/HumanSignal/label-studio-enterprise/commit/fc0daaea9fef1f9b3a5df8e868d39a8b832a57c1) also need to happen here?

Reply:
No since we're calling it after calling delete on each model run. I wanted to call it once versus after each object deletion. If we have another flow of deleting model runs later on we will need to add it there as well. For now we're good

Reply:
ah, makes sense